MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -324,6 +325,33 @@ public RasterizerState RasterizerState
             }
         }
 
+        /// <summary>
+        /// Blend factor for alpha blending.
+        /// </summary>
+        public Color BlendFactor
+        {
+            get { return _blendFactor; }
+            set
+            {
+                if(_blendFactor == value)
+                    return;
+
+                _blendFactor = value;
+
+                var newBlendState = _blendState.Clone();
+
+                newBlendState.BlendFactor = value;
+
+                newBlendState.BindToGraphicsDevice(this);
+
+                _blendState = newBlendState;
+
+                _actualBlendState = newBlendState;
+
+                _blendStateDirty = true;
+            }

```

Comment:
Also probably won't hurt to track `_blendFactorDirty` separately as often the factor is white and does not change.  This removes an extra API call for each blend state change.